pysat/instruments/methods/testing.py

```diff
@@ -515,3 +518,84 @@ def define_range():
                  'angle': [0.0, 2.0 * np.pi]}
 
     return def_range
+
+
+def create_files(inst, start, stop, freq=None, use_doy=True, root_fname=None,
```

Comment:
```suggestion
def create_files(inst, start, stop, freq='1D', use_doy=True, root_fname=None,
```

Reply:
Thanks! I'll try to remember to look out for the `=None` defaults.